### PR TITLE
Sarkars/merge r013

### PR DESCRIPTION
**Pull request by sayantan-nervana**

### CODE REVIEW

File: python/setup.in.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def get_tag(self):
     package_data= package_data_dict,
     cmdclass={'bdist_wheel': BinaryBdistWheel},
     extras_require={
+        'plaidml': ["tensorflow==1.13.0"],
```
Comment (sayantan-nervana) [inline python/setup.in.py]: TODO check: now that master is at 1.14, this will not work?

File: test/ci/buildkite/ngtf-cpu_centos-grappler.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@
       python3 build_ngtf.py --use_grappler --artifacts /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID
       
     label: ":hammer_and_wrench: Build"
-    timeout_in_minutes: 40
+    timeout_in_minutes: 60
```
Comment (sayantan-nervana) [inline test/ci/buildkite/ngtf-cpu_centos-grappler.yaml]: Increasing buildtimes, as CI seems to timeout during building with the new upgrade to tf1.14

File: test/opexecuter.h [hunk 3]
```diff
@@ -89,8 +89,6 @@ class OpExecuter {
 
   shared_ptr<ng::Function> get_ng_function() { return ng_function; }
 
-  // TODO(malikshr) : Overload RunTest() to take in tolerance
-
```
Comment (sayantan-nervana) [inline test/opexecuter.h]: This TODO is now a BEENDONE

File: python/setup.in.py (hunk 1 above)
Comment (avijit-nervana) [reply python/setup.in.py]: It should. Will find out :)

**Pull request merged**